packages/electron/README.md

```diff
@@ -45,19 +45,44 @@ Upgrading `electron` involves more than just bumping this package's `package.jso
     - The Ubuntu images in [base-internal](https://github.com/cypress-io/cypress-docker-images/tree/master/base-internal) are updated to be used in the [system binary tests](../../system-tests/test-binary) if any of the following are true for the images used inside the system binary tests:
       - The last two major [Ubuntu LTS Releases](https://ubuntu.com/about/release-cycle) are out-of-date.
       - The [NodeJS](https://nodejs.org/en) version is not the active LTS.
+- [ ] **Update `workflows.yml`**
+    - [ ] Ensure it references the new `base-internal` and `browsers-internal` Docker images
+    - [ ] Ensure the new Electron version is used as a build target in the `Build better-sqlite3 for CentOS 7` step
+    - [ ]
```

Comment:
```suggestion
```

It looks like this line should be removed.

Reply:
Thanks, Mike! Still making sure all of the rest of things are working as expected, so this file will likely change some more before it's marked ready for review.

Reply:
Sure. I'll hold back until the PR is marked for review.